test-data/unit/fixtures/bytes.pyi

```diff
@@ -0,0 +1,20 @@
+class Any: pass
```

Comment:
This is the builtins module used by the tests in check-bytes-str-unicode-python2.test.